### PR TITLE
Cleanup & improve 3rd party component's dashboards

### PR DESCRIPTION
**Pull request by francoisferrand**
Issue: ZENKO-3799

Screenshot 2021-12-30 at 23 03 31
Screenshot 2022-01-03 at 14 49 46
Screenshot 2021-12-30 at 23 04 40

### CODE REVIEW

**Comment by miniscruff** [comment]
Small thing here but i would split hits and misses. Because they will likely have very different values it can be hard to see any significant changes in the smaller one.

File: monitoring/dashboards/redis.json [hunk 1]
```diff
@@ -61,6 +61,66 @@
...
+                    "values": false
+                },
+                "text": {},
+                "textMode": "auto"
+            },
+            "pluginVersion": "8.0.6",
+            "targets": [
+                {
+                    "exemplar": true,
+                    "expr": "sum(up{namespace=\"zenko\", job=\"artesca-data-base-cache-metrics\"})",
```
Comment (lavigne958) [inline monitoring/dashboards/redis.json]: In this file the label `job` is an input value, we should replace the constant string with the variable `${job}`.

File: monitoring/dashboards/redis.json [hunk 2]
```diff
@@ -148,6 +208,66 @@
...
+                "values": false
+                },
+                "text": {},
+                "textMode": "auto"
+            },
+            "pluginVersion": "8.0.6",
+            "targets": [
+                {
+                "exemplar": true,
+                "expr": "sum(redis_db_keys{namespace=\"zenko\", job=\"artesca-data-base-cache-metrics\"})",
```
Comment (lavigne958) [inline monitoring/dashboards/redis.json]: same here we should replace the constant for job with the variable `${job}`.

**Review by lavigne958 — APPROVED** [review]

**Review by jbertran — APPROVED** [review]

**Pull request merged**